.github/workflows/build.yml

```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
```

Comment:
Why did we set specific version for runner?

Reply:
İf we use the latest version we get the following error:

Version 3.6.13 with arch x64 not found
The list of all available versions can be found here: https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json